src/DotnetVersions.props

```diff
@@ -3,7 +3,7 @@
 
   <PropertyGroup>
     <ScannerNetStandardVersion>netstandard2.0</ScannerNetStandardVersion>
-    <ScannerNetCoreVersion>netcoreapp2.1;netcoreapp3.1</ScannerNetCoreVersion>
```

Comment:
dotnet 2.1 was detected as vulnerable so I had to update to the latest security update (2.1.30)

Reply:
Does this change the minimum version the user must have installed to run analysis? If so, you might want to check with Tom since this is effectively a breaking change.

Also, does it matter if we are still targeting 2.1? The end user can decide which framework version to install i.e. if the user wants to be secure they can install 2.1.30 and the scanner will still work.

Reply:
I think you're right on both points. I've left this as a draft since I'm also not sure about the changes. I'll add a note for the next scanner meeting to discuss this.